package.json

```diff
@@ -31,6 +31,7 @@
     "sync-schema": "rm -rf data && mkdir data && graphql-fetch-schema https://metaphysics-staging.artsy.net -o data",
     "test:smoke": "scripts/smoke_test.sh",
     "test": "scripts/test.sh",
+    "test:jest": "node node_modules/.bin/jest",
```

Comment:
@eessex - we can no longer run `yarn jest some-file-name.tsx` from the command line in a common TDD fashion, since it now contains `JEST_JUNIT_OUTPUT` and `runInBand`. I added this temporary command but we should make sure that `yarn jest` points directly at the jest bin and doesn't contain any intermediary CI-centric config.